### PR TITLE
Add Trash Can

### PR DESCRIPTION
**Pull request by drahenshaw**
Added Trash Can recipe and image. A personal favorite!

### CODE REVIEW

File: src/recipes/trash-can.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    {
+      "quantity": "1",
+      "measure": "can",
+      "ingredient": "Red Bull"
+    }
+  ],
+  "directions": [
+    "Fill large glass with ice. Add alcohol ingrediants. Open the Red Bull and place upside down in the glass."
+  ],
+  "image": "vodka-soda.jpg",
```
Comment (alfg) [inline src/recipes/trash-can.json]:
```suggestion
  "image": "trash-can.jpg",
```
Also, update the image filename to `trash-can.jpg` too. Thanks!

Comment (drahenshaw) [reply src/recipes/trash-can.json]: Oops, sorry about that! It should be resolved now! Thank you!

File: src/recipes/trash-can.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Tash Can",
```
Comment (alfg) [inline src/recipes/trash-can.json]:
```suggestion
  "name": "Trash Can",
```

**Comment by alfg** [comment]
Thanks for the quick fixes. That damn R key! :)

**Pull request merged**